generators/typescript-v2/dynamic-snippets/src/__test__/__snapshots__/DynamicSnippetsGenerator.test.ts.snap

```diff
@@ -237,10 +237,6 @@ async function main() {
     const client = new AcmeClient();
     await client.service.post({
         file: new Blob(["Hello, world!"]),
-        fileList: [
-            new Blob(["First"]),
-            new Blob(["Second"]),
-        ],
```

Comment:
I'm surprised by this one since this should just use the `@fern-api/typescript-ast` package. If the change isn't obvious, I can take a closer look in a follow-up.